### PR TITLE
Chat channel list

### PR DESCRIPTION
**Pull request by boennemann**
Add description for all of our channels.

Merge https://github.com/hoodiehq/hood.ie/pull/168 first.

### CODE REVIEW

**Comment by janl** [comment]
+1 on the chat page wording

File: chat/index.html [hunk 1]
```diff
@@ -23,12 +23,22 @@
 </div>
 <div class="cb">
     <article class="grid-9">
-        <h2>Now accessible via Slack</h2>
+        <h2>Send and receive the same messages on IRC and Slack.</h2>
         <p>
-          IRC isn’t the most user-friendly chat system. <a href="https://slack.com">Slack</a> on the other hand is a very user-friendly chat system. We have set up an <a href="http://rauchg.com/slackin/" title="&raquo; Slackin">IRC to Slack bridge</a> (that we generously borrowed from Socket.io, thank you!).
+          IRC is an open and very common, but not the most user-friendly chat system. Slack on the other hand is a very user-friendly one. We have developed an <a href="https://github.com/hoodiehq/slack-irc-duplex" title="slack-irc-duplex on GitHub">IRC/Slack duplexer</a>, so you can choose whatever you personally prefer.
```
Comment (varjmes) [inline chat/index.html]: "IRC is an open and very common" doesn't make sense.
Should it be "IRC is open and very common"? :)

Comment (varjmes) [inline chat/index.html]: I'm not really sure what "duplexer" means if I saw that for the first time. Is there scope to add a line along the lines of "everything you post to the IRC channel is sent to Slack and vice-versa"?

**Comment by zoepage** [comment]
+1 <3

**Comment by christophwitzko** [comment]
nice work, LGTM :+1:

File: chat/index.html (hunk 1 above)
Comment (davidpfahler) [inline chat/index.html]: The sentence is actually correct. The main sentence is: `IRC is an open and very common chat system`. There is a comma missing, which is probably what confused you. Solution:

`IRC is an open and very common, but not the most user-friendly, chat system.`

or

`IRC is an open and very common – but not the most user-friendly – chat system.`

But maybe we just shouldn't be so clever an just say

`IRC is open and very common, but Slack is more user-friendly.`

saving an entire sentence in the process ;)

Comment (HipsterBrown) [inline chat/index.html]: I think the whole phrase is "IRC is a very open and common, but not the most user-friendly, chat system." :)

I agree with clarifying what the "duplex" system does exactly.

Comment (varjmes) [inline chat/index.html]: ah, thank you for the clarification @davidpfahler :)

**Comment by boennemann** [comment]
Incorporated the feedback @Charlotteis @HipsterBrown @davidpfahler -> https://github.com/hoodiehq/hood.ie/commit/c00d7239c692fa13d47afd18543cdf4e0d9b7f24

**Comment by varjmes** [comment]
@boennemann looks great to me!

**Comment by davidpfahler** [comment]
@boennemann +1 for all changes

**Comment by varjmes** [comment]
I am happy to merge this, but can't do so until https://github.com/hoodiehq/hood.ie/pull/168 is merged. Would you like to remove the commits that are already covered in #168, @boennemann, or wait until that PR is merged?

**Comment by boennemann** [comment]
@janl @Charlotteis @zoepage 

Made this PR independent of #168, please merge now :)

**Comment by varjmes** [comment]
Thank you, @boennemann :)

**Comment by boennemann** [comment]
@Charlotteis Thanks :tada: 

@christophwitzko May I now hint you towards #178? :)

**Pull request merged**